### PR TITLE
Fix typescript tests

### PR DESCRIPTION
**Pull request by cocreature**
The new version of eslint has broken stuff, see the linked issue.

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: language-support/ts/codegen/tests/src/DA/Test/Daml2js.hs [hunk 1]
```diff
@@ -26,8 +26,9 @@ eslintVersion :: T.Text
 eslintVersion = "^6.8.0"
 
 -- Version of typescript-eslint for linting the generated code.
+-- 2.23 produces an error https://github.com/typescript-eslint/typescript-eslint/issues/2009
```
Comment (rohanjr) [inline language-support/ts/codegen/tests/src/DA/Test/Daml2js.hs:29]: This should actually say 2.32

Comment (rohanjr) [reply language-support/ts/codegen/tests/src/DA/Test/Daml2js.hs:29]: I can fix in a followup to get this one through

Comment (cocreature) [reply language-support/ts/codegen/tests/src/DA/Test/Daml2js.hs]: just make it a suggestion and you save me having to open an editor

Comment (rohanjr) [reply language-support/ts/codegen/tests/src/DA/Test/Daml2js.hs]: i would have taken care of it ;)

**Pull request merged**